trustregistry/migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
This can either be removed or just edited with a one line summary of what this directory is for